k8s/namespaces/xui/xui-webapp-multiday-hearings-integration/demo.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         SERVICES_IDAM_ISS_URL: https://forgerock-am.service.core-compute-idam-demo.internal:8443/openam/oauth2/realms/root/realms/hmcts
         DEBUG: xuiNode:*
         SERVICES_IDAM_OAUTH_CALLBACK_URL: oauth2/callback
-        STUB: false
+        STUB: true
```

Comment:
shouldn't change this value it will make the share case running on mock api, should add or remove a dummy variable.